packages/zoe/src/contractFacet/zcfZygote.js

```diff
@@ -31,6 +31,22 @@ import { InvitationHandleShape } from '../typeGuards.js';
 
 const { Fail } = assert;
 
+/**
+ * Wrap getTerms so it checks the fit of the custom terms
+ *
+ * @param {InstanceRecordManagerGetTerms} getTerms
+ * @param {Pattern} customTermsShape
+ */
+function fitGetTerms(getTerms, customTermsShape) {
```

Comment:
```suggestion
const fitGetTerms = (getTerms, customTermsShape) => {
```

Reply:
`function` has a lot of additional semantics, none of which are relevant here:
   * It is constructible. `new fitGetTerms(...)` invokes its construct behavior
   * `fitGetTerms.prototype` exists and points at a new empty object (in order to support that construct behavior)
   * its initialization is hoisted, most dramatically for exports (not relevant here), making the unfrozen value observable before an immediately following `harden` in the exporting module can protect it.

Reply:
Will do and force push. My fingers tend not to use `function` anymore but I used "extract function" in my IDE and didn't notice.